app/src/androidTest/java/com/github/geohunt/app/mocks/MockChallenge.kt

```diff
@@ -5,9 +5,9 @@ import com.github.geohunt.app.model.database.api.Location
 import java.time.LocalDateTime
 
 fun MockChallenge(
-    id: String = "",
-    authorId: String = "",
-    photoUrl: String = "",
+    id: String = "1",
+    authorId: String = "1",
+    photoUrl: String = "https://picsum.photos/200",

```

Comment:
In general try not using external url from within the test context, because this means that something external to the test environment can impact tests.

Reply:
You can instead use a url to a local image of the emulator such as `http://10.0.2.2:9199/geohunt-1.appspot.com/images/challenges-images.png`

Reply:
To be honest, I am not sure whether this will work. I will simply remove that URL because I am definitely tired.